src/tab/Tabs.stories.ts

```diff
@@ -0,0 +1,442 @@
+import { within } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+import * as jest from 'jest-mock';
+import { html, nothing } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import expect from '../utils/ExpectDOM.js';
+import { assignToSlot, ComponentStoryFormat, CSFIdentifier, querySelectorAsync, raw, getSourceFromLit } from '../utils/StoryUtils.js';
+
+import { Tab } from './Tab.js';
+import { TabHeader } from './TabHeader.js';
+import { Tabs } from './Tabs.js';
+import '../label/Label.js';
+import './TabHeader.js';
+import './Tab.js';
+import './Tabs.js';
+import '../icon/Icon.js';
+
+export default {
+    title: 'UI Components/Tabs',
+    component: 'omni-tabs',
+    argTypes: {
+        header: {
+            control: 'text'
+        }
+    }
+} as CSFIdentifier;
+
+interface Args {
+    header: string;
+    '[Default Slot]': string;
+}
+
+export const Interactive: ComponentStoryFormat<Args> = {
+    render: (args: Args) => html`
+    <omni-tabs data-testid='test-tabs'>
+        ${args.header ? html`${'\r\n'}${unsafeHTML(assignToSlot('header', args.header))}` : nothing}
+        ${args['[Default Slot]'] ? html`${'\r\n'}${unsafeHTML(args['[Default Slot]'])}` : nothing}
+    </omni-tabs>
+`,
+    frameworkSources: [
+        {
+            framework: 'Vue',
+            load: (args) =>
+                getSourceFromLit(Interactive!.render!(args), undefined, (s) =>
+                    s.replace(' active', ' :active="true"').replace(' disabled', ' :disabled="true"')
+                )
+        }
+    ],
+    name: 'Interactive',
+    description: () => html`
+        <p>
+        The <code class="language-html">&lt;omni-tabs&gt;</code> component will display content based on the nested <code class="language-html">&lt;omni-tab&gt;</code> component(s). 
+        </p>
+        <p>
+        Tab headers are rendered by either setting the <code>header</code> attribute of the <code class="language-html">&lt;omni-tab&gt;</code> component or via slotted <code class="language-html">&lt;omni-tab-header&gt;</code> component(s) that targets the <code class="language-html">&lt;omni-tabs&gt;</code> header slot. 
+        </p>
+    `,
+    args: {
```

Comment:
The string contained in `args` for the default slot has single quotes for attribute values. Should be double quotes for consistency.

Reply:
Updated as part of the Tab components story file split